pages/admin/index.tsx

```diff
@@ -49,23 +57,24 @@ function renderStatusBadge({ value }: StatusProps) {
 type NameProps = {
   readonly value: {
     name: string;
-    id: string;
+    rcdUserId: string;
   };
 };
 
 function renderName({ value }: NameProps) {
```

Comment:
Similar to in @EmilioMena 's PR, I think we can just put all of these render functions in the columns configuration to keep it all in one place. Also, I believe it can infer the types if you do it like that.